archiver.go

```diff
@@ -26,6 +27,18 @@ type Archiver interface {
 // SupportedFormats contains all supported archive formats
 var SupportedFormats = map[string]Archiver{}
 
+// WindowsReplacer replaces invalid characters in Windows filenames with underscore
+var WindowsReplacer = strings.NewReplacer(
+	"<", "_",
+	">", "_",
+	":", "_",
+	"\"", "_",
+	"/", "_",
+	"?", "_",
+	"*", "_",
+	"|", "_",
+)
```

Comment:
- I'm not sure this needs to be exported.

- We could just remove the character entirely, rather than replacing it.

- This will introduce filename collisions, for example `Hello?.txt` and `Hello:.txt` will become the same file. That might cause extractions to fail.